### PR TITLE
[RDAPI-22391] API Manager REST APIs, JVM flags to import Swagger/OAS APIs with external references.

### PR DESCRIPTION
**Pull request by moscaribera**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Swagger JSON or YAML format in register a back-end REST API in API Manager.
API Manager REST APIs, JVM flags to import Swagger/OAS APIs with external references.
Revert warning on import API Manager REST API.

## Deploy preview link

- https://deploy-preview-2213--axway-open-docs.netlify.app/docs/apim_administration/apimgr_admin/api_mgmt_workflow/#register-a-back-end-rest-api-in-api-manager
- https://deploy-preview-2213--axway-open-docs.netlify.app/docs/apim_administration/apimgr_admin/api_mgmt_register_web/#import-an-existing-back-end-rest-api
- https://deploy-preview-2213--axway-open-docs.netlify.app/docs/apim_reference/api_mgmt_rest_api/#import-the-api-manager-rest-api

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by andreamussap** [comment]
@moscaribera please fix conflict in this PR

**Comment by moscaribera** [comment]
Conflict fixed.

File: content/en/docs/apim_reference/api_mgmt_rest_api.md [hunk 1]
```diff
@@ -28,16 +28,7 @@ You can import the API Manager REST API Swagger 2.0 (OAS2) or OAS3 definitions i
     * **Organization**: Select the organization from the list (for example, **API Development**).
 4. Click **Import** to import the API Manager API.
 
-{{< alert title="Note" color="primary" >}}SmartBear Swagger Parser third-party libraries are used to import OAS definitions. Certificate checking is used in the parser when resolving remote references, therefore the following flags might need to be added to `jvm.xml` file to resolve external references:
```
Comment (andreamussap) [inline content/en/docs/apim_reference/api_mgmt_rest_api.md:31]: @moscaribera Just to double check. Did you delete the note that was added to this page via PR #2211  and moved the note to another page?

Comment (moscaribera) [reply content/en/docs/apim_reference/api_mgmt_rest_api.md:31]: Yes. This page talks especifically about importing the API Manager Swagger/OAS API definition whereas the note is intended for importing any Swagger/OAS API definition.

**Pull request merged**